diff --git a/lib_pypy/pypy_tools/build_cffi_imports.py b/lib_pypy/pypy_tools/build_cffi_imports.py
index 97c9722c4b..876bf4c4da 100644
--- a/lib_pypy/pypy_tools/build_cffi_imports.py
+++ b/lib_pypy/pypy_tools/build_cffi_imports.py
@@ -302,6 +302,8 @@ def create_cffi_import_libraries(pypy_c, options, basedir, only=None,
 
         print('*', ' '.join(args), file=sys.stderr)
         if embed_dependencies and key in cffi_dependencies:
+            print("Nixpkgs: skipping fetching/building dependency", key)
+        elif False:
             status, stdout, stderr = _build_dependency(key, env=env)
             if status != 0: # or key in ("_ssl"):
                 print("stdout:")
