diff --git a/topaz/utils/packing/floatpacking.py b/topaz/utils/packing/floatpacking.py
index 7d71da25..66d33ca4 100644
--- a/topaz/utils/packing/floatpacking.py
+++ b/topaz/utils/packing/floatpacking.py
@@ -1,3 +1,5 @@
+import math
+
 from rpython.rlib import rfloat
 from rpython.rlib.rstruct.ieee import float_pack
 
@@ -20,7 +22,7 @@ def make_float_packer(size, bigendian):
             try:
                 unsigned = float_pack(doubleval, size)
             except OverflowError:
-                unsigned = float_pack(rfloat.copysign(rfloat.INFINITY, doubleval), size)
+                unsigned = float_pack(math.copysign(rfloat.INFINITY, doubleval), size)
             for i in xrange(size):
                 l[i] = chr((unsigned >> (i * 8)) & 0xff)
             if bigendian:
diff --git a/topaz/utils/regexp.py b/topaz/utils/regexp.py
index 0ac11454..d9d99b50 100644
--- a/topaz/utils/regexp.py
+++ b/topaz/utils/regexp.py
@@ -3,14 +3,14 @@ import sys
 from rpython.rlib.listsort import make_timsort_class
 from rpython.rlib.objectmodel import specialize
 from rpython.rlib.rstring import StringBuilder
-from rpython.rlib.rsre.rsre_core import (
+from rpython.rlib.rsre.rsre_constants import (
     OPCODE_LITERAL, OPCODE_LITERAL_IGNORE, OPCODE_SUCCESS, OPCODE_ASSERT,
     OPCODE_MARK, OPCODE_REPEAT, OPCODE_ANY, OPCODE_ANY_ALL, OPCODE_MAX_UNTIL,
     OPCODE_MIN_UNTIL, OPCODE_GROUPREF, OPCODE_AT, OPCODE_BRANCH, OPCODE_RANGE,
     OPCODE_JUMP, OPCODE_ASSERT_NOT, OPCODE_CATEGORY, OPCODE_FAILURE, OPCODE_IN,
     OPCODE_NEGATE, OPCODE_GROUPREF_EXISTS
 )
-from rpython.rlib.rsre.rsre_core import (
+from rpython.rlib.rsre.rsre_constants import (
     AT_BEGINNING, AT_BEGINNING_LINE, AT_BEGINNING_STRING, AT_BOUNDARY,
     AT_NON_BOUNDARY, AT_END_LINE, AT_END_STRING,
 )
